gui/freeadmin/static/lib/js/freeadmin.js

```diff
@@ -2729,9 +2729,9 @@ require([
         dialog.show();
     };
 
-    editObject = function(name, url, nodes, onload) {
+    editObject = function(name, url, nodes, onload, id="edit_dialog") {
```

Comment:
I am skeptical this will work at all in all browsers (id="foo" like python)

Also I find it too critical to change that code at this stage, can you use commonDialog directly instead of editObject, please?